src/components/update-reminder/update-reminder-content.js

```diff
@@ -1,27 +1,32 @@
 export const updateContent = {
-  date: '17 April 2024',
+  date: '22 May 2024',
   features: [
     {
-      title:
-        'Enable stateful URLs with node filters and expand/collapse modular pipelines',
+      title: 'Add "Expand All Pipelines" toggle to the utility bar',
```

Comment:
If this is worth including in the update reminder, should we include it in the "Major features and improvements" section of the release notes?